### PR TITLE
{lib}[foss/2022a] segmentation-models-pytorch v0.3.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/39208b2c06584fca815593578d0c947e for a full test report.

File: easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    ('efficientnet_pytorch', '0.7.1', {
+        'checksums': ['00b9b261effce59d2d47aae2ad238c29a2a65175470f41ada7ecac439b7c1ee1'],
+    }),
+    ('huggingface_hub', '0.15.1', {
+        'checksums': ['a61b7d1a7769fe10119e730277c72ab99d95c48d86a3d6da3e9f3d0f632a4081'],
+    }),
+    ('timm', '0.9.2', {
+        'checksums': ['d0977cc5e02c69bda979fca8b52aa315a5f2cb64ebf8ad2c4631b1e452762c14'],
+    }),
+    ('tqdm', '4.64.0', {
```
Comment (smoors) [inline easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb:36]: please use `tqdm/4.64.0-GCCcore-11.3.0` dependency instead?

File: easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        'checksums': ['40be55d30e200777a307a7585aee69e4eabb46b4ec6a4b4a5f2d9f11e7d5408d'],
+    }),
+    ('six', '1.16.0', {
```
Comment (smoors) [inline easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb:39]: `six` is already part of the Python module, so can be removed

File: easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb (hunk 1 above)
Comment (WilleBell) [reply easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb]: tqdm is a dependency on the new commit

File: easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb (hunk 2 above)
Comment (WilleBell) [reply easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb]: six was removed on the new commit

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @WilleBell!

**Pull request merged**